src/modules/assessment/services/assessment.service.ts

```diff
@@ -136,11 +137,18 @@ export class AssessmentService {
      * @param assessmentId
      * @returns
      */
-    async getFullAssessment(assessmentId: number): Promise<FullAssessment> {
+    async getFullAssessment(assessmentId: number, uuid: string): Promise<FullAssessment> {
+        const query = {
+            id: assessmentId,
+            isActive: true,
+            uuid,
+        }
         const assessment: FullAssessment = (await this.assessmentRepository.findOne(
-            assessmentId,
-            { relations: ['clinician', 'patient'] },
+            {
+                where: [query, { ...query, uuid: IsNull() }], relations: ['clinician', 'patient']
+            },
         )) as FullAssessment;
+        // console.log(assessment)
```

Comment:
Not sure if the console log comment here is needed. Can apply standard nestjs logger, if logging is necessary at this step, or omit it.